<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="rosbag" default="true" />
  <param name="use_sim_time" value="$(arg rosbag)" />
  <arg name="queue_size" default="100" />

  <node name="edge_filter_compressed" pkg="rtabmap_create_data" type="convert_compressed_image.py" />

  <include file="$(find rtabmap_ros)/launch/rtabmap.launch">
    <arg name="rgb_topic" value="/camera/color/decode_image_raw" />
    <arg name="depth_topic" value="/camera/aligned_depth_to_color/image_raw" />
    <arg name="camera_info_topic" value="/camera/color/camera_info" />
    <arg name="imu_topic" value="/rtabmap/imu" />
    <arg name="approx_sync" value="false" />
    <arg name="wait_imu_to_init" value="true" />
    <arg name="args" value="--delete_db_on_start --Optimizer/GravitySigma 0.3" />
  </include>
  <node name="rosbag_player" pkg="rosbag" type="play"
    args="--clock /home/aisl/catkin_ws/2022-10-13-05-15-46.bag" />
</launch>